AirCasting/CreateSessionViews/ABConnectedView.swift

```diff
@@ -14,6 +14,7 @@ struct ABConnectedView: View {
     @EnvironmentObject var userAuthenticationSession: UserAuthenticationSession
     @EnvironmentObject var sessionContext: CreateSessionContext
     @Binding var creatingSessionFlowContinues : Bool
+    let baseURL: BaseURLProvider
```

Comment:
can this be `private`?

Reply:
I cannot to this or at least I am not able to because than preview has some problem esp. "'ABConnectedView' initializer is inaccessible due to 'private' protection level"

Reply:
Note I only point those `private`s out, so if it's not possible just don't do it :)